cognite/neat/rules/importers/_dms2rules.py

```diff
@@ -179,21 +180,12 @@ def to_rules(
 
     def _create_rule_components(
         self,
+        data_model: dm.DataModelApply,
         schema: DMSSchema,
```

Comment:
would not schema have data model ? why it is necessary to additionally data_model as arg?